frontend/pages/organizations/create.vue

```diff
@@ -117,6 +118,8 @@
 
 <script setup lang="ts">
 import type { OrganizationCreateFormData } from "~/types/entities/organization";
+import {Toaster, toast } from 'vue-sonner'
+import BtnAction from "~/components/btn/action/BtnAction.vue";
```

Comment:
Quick note that we don't need to import the component in Nuxt, but then I think this was likely an auto import? :)

Reply:
Yes, I didn't even realize that it was added to the imports 😄.